app/code/Magento/CustomerGraphQl/Model/Resolver/ResetPassword.php

```diff
@@ -118,7 +118,7 @@ public function resolve(
                 $args['newPassword']
             );
         } catch (LocalizedException $e) {
-            throw new GraphQlInputException(__('Cannot set the customer\'s password'), $e);
+            throw new GraphQlInputException(__('The password must be at least 8 characters long, minimum of 3 different classes of characters: Lower Case, Upper Case, Digits, Special Characters.'), $e);
```

Comment:
@sudheers-kensium we try not add new phrases. $e->getMessage() should bubble up the exceptions from the api, which will be more meaningful and is the scope of this bug. $this->customerAccountManagement->resetPassword throws Input Exceptions, you can catch them in the resolver to give meaningful error responses back. For example, I see a bunch of meaningful validations here \Magento\Customer\Model\AccountManagement::checkPasswordStrength
Once you're able to get those messages, you can cover a couple in api functional tests, just to make sure the validation errors are being thrown.